public/video-ui/src/components/VideoEdit/formComponents/VideoCategoryId.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class VideoCategoryIdEdit extends React.Component {
+
+  onUpdateCategoryId = (e) => {
+
+    let newMetaData = Object.assign({}, this.props.video.data.metadata, {
```

Comment:
`categoryId` is now `youtubeCategoryId` (more descriptive, there was some confusion with the reguar `category` field). And is no longer nested inside `data.metadata`